lib/src/extensions.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:ui';
+
+import 'package:path_drawing/path_drawing.dart';
+
+extension PathDrawingExtension on Path {
+  Path trim(
+    double percentage, {
+    bool firstOnly = true,
+    PathTrimOrigin origin = PathTrimOrigin.begin,
+  }) {
+    return trimPath(this, percentage, firstOnly: firstOnly, origin: origin);
+  }
+
+  Path dash({
+    required CircularIntervalList<double> dashArray,
+    DashOffset? dashOffset,
+  }) {
+    return dashPath(this, dashArray: dashArray, dashOffset: dashOffset);
+  }
+}
+
+extension ParsePathExtension on String {
```

Comment:
I don't think this is something we want. It's somehow invasive and not even clear what it means. Even if we keep it, we should turn it into a method `toPath()` instead, as being a getter here is misleading.